src/components/Questions/QuestionModal.vue

```diff
@@ -143,7 +144,10 @@ export default defineComponent({
     }
 
     function navigateToQuestion(questionIndex: number) {
-      state.localCurrentQuestionIndex = questionIndex
+      if (!isQuestionFetched(questionIndex)) {
```

Comment:
I might be missing something basic; but if the `questionIndex` starts with 0, shouldn't this be incremented 1? Or is it incremented before calling?

If so, in the `showNextQuestion` method, why is it index + 1?